### ISSUE TITLE
KopiaUI should not be needed to run in the background for scheduled snapshots

### ISSUE THREAD

**Issue by rpgdev** [kopia/kopia#2716]
Considering kopia is used in the background to do the snapshotting it's insanely wasteful to have the user perpetually run a electronjs (or similar) UI app in the background just so the snapshots are performed on schedule when this doesn't need any UI interaction most of the time. At least for Unix-like systems this should not be necessary when cron comes standard.

**Comment by andriej** [comment]
I agree - also kopiaUI keeps connection to repository even when there's no maintenance nor snapshot tasks anytime soon - as described in: https://kopia.discourse.group/t/kopia-maintains-connection-to-repository/1732

Having service in-between for non-cli users of kopia would be nice (at least in documentation maybe, via some setup-script)?
